locust/stats.py

```diff
@@ -467,7 +497,7 @@ def print_stats(stats):
     except ZeroDivisionError:
         fail_percent = 0
 
-    console_logger.info((" %-" + str(STATS_NAME_WIDTH) + "s %7d %12s %42.2f") % ('Total', total_reqs, "%d(%.2f%%)" % (total_failures, fail_percent), total_rps))
+    console_logger.info((" %-" + str(STATS_NAME_WIDTH) + "s %7d %12s %7d %7d %7d %10d %7.2f") % ('Total', total_reqs, "%d(%.2f%%)" % (total_failures, fail_percent), total_average, total_min, total_max, total_median, total_rps))
```

Comment:
Again, in the case of exiting before executing, total_min will be None and this line will fail with a TypeError.